src/api.v2/helpers/pipeline/pipelineConstruct/constructors/assignPodToPipeline.js

```diff
@@ -43,7 +43,7 @@ const requestPod = (context, step) => {
     Type: 'Task',
     Resource: 'arn:aws:states:::sns:publish',
     Parameters: {
-      TopicArn: `arn:aws:sns:${config.awsRegion}:${accountId}:work-results-${environment}-${sandboxId}-v2`,
```

Comment:
Why do you remove the v2 from here? You have already created new topics for v2: https://github.com/hms-dbmi-cellenics/iac/pull/384/files

Reply:
This was reverted because pod assignment is still not yet implemented for v2. Pod assignment will be implemented in the following ticket to implement QC: https://biomage.atlassian.net/browse/BIOMAGE-1877